### PR TITLE
Rubocop added

### PR DESCRIPTION
**Pull request by blassoto**
### Summary

- Add rubocop to the project

### Trello Card

https://trello.com/c/CCpANJuK

### CODE REVIEW

File: app/channels/application_cable/channel.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (aleochoam) [inline app/channels/application_cable/channel.rb]: Let's make an exception to this magic comment in the rubocop file

File: Gemfile [hunk 2]
```diff
@@ -38,4 +39,4 @@ group :development do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (aleochoam) [inline Gemfile]: Why this EOF removal? maybe the editor remove it?

Comment (blassoto) [reply Gemfile]: Maybe I remove it

File: app/channels/application_cable/channel.rb (hunk 1 above)
Comment (blassoto) [reply app/channels/application_cable/channel.rb]: done!

**Comment by blassoto** [comment]
> Another comments, the commits are written in english, to keep code and commits in the same language and in the description of the PR, we add a summary of the funcionality and the link to the jira/trello card, can you update the description please?
> 
> An example:
> ### Summary
> 
>     * Add rubocop to the project
> 
> 
> ### Trello Card
> 
> http://www.example.org

**Comment by blassoto** [comment]
@aleochoam @franciscoabalan  done!

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Layout/LineLength:
+  Max: 99
+Style/HashEachMethods:
+  Enabled: true
+Style/HashTransformKeys:
+  Enabled: true
+Style/HashTransformValues:
+  Enabled: true
+Style/FrozenStringLiteralComment:
+  Enabled: false
```
Comment (aleochoam) [inline .rubocop.yml]: EOF

File: Gemfile [hunk 4]
```diff
@@ -28,6 +28,7 @@ gem 'puma', '~> 3.7'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'rubocop', require: false
```
Comment (aleochoam) [inline Gemfile]: We usually lock the gem version in order to keep them under control, the prefered way to do that is with the Pessimistic Operator, an example:

```
gem 'rubocop', '~> 0.59'
```

File: Gemfile (hunk 2 above)
Comment (aleochoam) [reply Gemfile]: Can you add it again please?

**Comment by blassoto** [comment]
Done!

**Pull request merged**